Sources/Defaults/Migration.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+extension Defaults {
+	public static func migration<Value: Defaults.Serializable & Defaults.NativeType>(_ keys: Key<Value>...) {
```

Comment:
```suggestion
	public static func migrate<Value: Defaults.Serializable & Defaults.NativeType>(_ keys: Key<Value>...) {
```

I think the function name should be more specific to what's being migrated. We might add other kinds of migrations in the future.

Reply:
How about `migrateFromCodableToNative`?